### PR TITLE
Allow out_dynamic_vars parameter to be used

### PR DESCRIPTION
**Pull request by lewisjared**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Description in ``CHANGELOG.rst`` added

Doesn't convert out_dynamic_vars to an integer

### CODE REVIEW

**Comment by lewisjared** [comment]
#254

**Comment by lewisjared** [comment]
It would be nice to have a beta.2 release after this as this is super handy for the api

File: pymagicc/core.py [hunk 1]
```diff
@@ -777,7 +777,9 @@ def _format_config(self, config_dict):
 
     def _convert_out_config_flags_to_integers(self, config_dict):
         for key, value in config_dict.items():
-            if key.startswith("out") and key != "out_ascii_binary":
+            # Ignore all out_x_vars variables as that should be a list of strings
+            # Note that there is a variable OUT_ALLOWANYDYNAMICVARS which should be in integer
```
Comment (znicholls) [inline pymagicc/core.py]: What's `OUT_ALLOWANYDYNAMICVARS`?

**Comment by znicholls** [comment]
> It would be nice to have a beta.2 release after this as this is super handy for the api

Can do

File: pymagicc/core.py (hunk 1 above)
Comment (lewisjared) [reply pymagicc/core.py]: Malte has added a variable to force the calculation of some of the summary variables. Previously these were only calculated when OUT_KEYDATA_X was set. This isn't in master yet

Comment (znicholls) [reply pymagicc/core.py]: ok so a discussion for the MAGICC repo then

**Comment by znicholls** [comment]
@lewisjared https://pypi.org/project/pymagicc/2.0.0b3/

**Pull request merged**